rosetta/text/vw_helpers.py

```diff
@@ -318,21 +318,18 @@ def _set_probabilities(self, topics, predictions):
         Others can be derived and appear as "properties"
         (in the decorator sense).
         """
-        topic_sums = topics.sum()
-        self.pr_topic = topic_sums / topic_sums.sum()
+        self._lambda_word_sums = topics.sum()
+        self.pr_topic = self._lambda_word_sums / self._lambda_word_sums.sum()
 
+        # tokens & topic
         word_sums = topics.sum(axis=1)
         self.pr_token = word_sums / word_sums.sum()
-        self.pr_topic_token = topics / topics.sum().sum()
+        self.pr_token_topic = topics / self._lambda_word_sums.sum()
+        self.pr_token_topic.index.name = 'token'
```

Comment:
topics has an index name 'hash_val' and columns are the topic names, as do all the data frame/series here. 

are you saying 
```python
self.pr_token_topic.index.name = 'token'
``` 
is superfluous, which I agree with and am removing